cli/tests/integration.rs

```diff
@@ -0,0 +1,12 @@
+use foundry_cli_test_utils::forgetest_external;
```

Comment:
this is now part of the default cargo test process.
I'd propose feature gating this so these are only run in CI and not locally when running `cargo test`

Reply:
Now gated under `integration-tests`